dspace-api/src/main/java/org/dspace/importer/external/crossref/CrossRefDateMetadataProcessor.java

```diff
@@ -0,0 +1,79 @@
+/**
+ * The contents of this file are subject to the license and copyright
+ * detailed in the LICENSE and NOTICE files at the root of the source
+ * tree and available online at
+ *
+ * http://www.dspace.org/license/
+ */
+package org.dspace.importer.external.crossref;
+
+import java.text.SimpleDateFormat;
+import java.util.ArrayList;
+import java.util.Collection;
+import java.util.Iterator;
+
+import com.fasterxml.jackson.core.JsonProcessingException;
+import com.fasterxml.jackson.databind.JsonNode;
+import com.fasterxml.jackson.databind.ObjectMapper;
+import org.apache.logging.log4j.LogManager;
+import org.apache.logging.log4j.Logger;
+import org.dspace.importer.external.metadatamapping.contributor.JsonPathMetadataProcessor;
+import org.joda.time.LocalDate;
+
+/**
+ * This class is used for CrossRef's Live-Import to extract
+ * issued attribute.
+ * Beans are configured in the crossref-integration.xml file.
+ * 
```

Comment:
FYI this comment introduces trailing whitespace.

Reply:
Hi @alanorth, I removed the trailing whitespace from the javadoc. Let me know, thank you!

Reply:
Thanks @francescopioscognamiglio. I don't know if anyone has any other comments about the code, but it works as expected.